src/app/header/registration/registration.component.ts

```diff
@@ -43,10 +46,20 @@ export class RegistrationComponent implements OnInit {
   }
 
 
-  registrationSubmit() {
-    console.log(this.registrationForm)
+  registrationSubmit(  firstName: string, lastName: string, email: string, password: string, passwordConfirm: string, city: string, street: string, zipCode: string,) {
```

Comment:
Zamiast pobierać wszystkie argumenty z templatki można pobierać w funkcji bezpośrednio z FormGroup, na zasadzie:
 `const fullForm : RegForm = {firstName: this.registrationForm.getControl('firstname').value, lastName: this.registrationForm.getControl('lastname').value` itd...

Reply:
To co napisałem to po prostu inny sposób -> Twoje też jest ok.